tests/test_models.py

```
@@ -373,19 +467,43 @@ def test_transformer_decoder():
+        decoder_queries,
+        embedding_map,
+        boxes,
+        times,
+        enc_boxes,
+        enc_times,
+    )
 
     assert decoder_features.size() == decoder_queries.size()
 
     # with position
     pos_emb = query_pos_emb = torch.ones_like(encoder_features)
```

Ruff

Local variable `query_pos_emb` is assigned to but never used

Remove assignment to unused variable `query_pos_emb`

(F841)